Source/WTF/wtf/SystemTracing.h

```diff
@@ -150,6 +150,8 @@ enum TracePointCode {
     SyntheticMomentumEvent,
     RemoteLayerTreeScheduleRenderingUpdate,
     DisplayLinkUpdate,
+    UpdateLayerContentBuffersStart,
```

Comment:
These shouldn't be here but in the `GTKWPEPortRange` below, otherwise you need to update the `Tools/Tracing/SystemTracePoints.plist` file, to make sure it won't break the Apple ports, once they add another enum entry below this one. So better move to our section.